### ISSUE TITLE
Implementation issue 

### ISSUE THREAD

**Issue by aimanmutasem** [chrisjbryant/errant#31]
Dear Chris :) 

I have applied Errant for fce.test.me and I got unexpected results as : 
```
=========== Span-Based Correction ============
TP      FP      FN      Prec    Rec     F0.5
2       15503   4547    0.0001  0.0004  0.0002
=======================================
```
I follow you implementation as same as in the documentation as follow: 

1. I have applied `errant_parallel` using, `errant_parallel -orig m2Scripts/orig_sentes.txt -cor m2Scripts/corec_sentes.txt -out m2Scripts/output.m2` files as in [GitHub](https://github.com/aimanmutasem/Apply-errant/tree/main/m2Scripts). Actually, I'm confused about the format of the `parallel corrected text file`. 

2. For the `errant_m2` I applied as `errant_m2 -auto m2Scripts/output.m2 -out m2Scripts/auto_output`.

3. The last step `errant_compare` as `errant_compare -hyp m2Scripts/auto_output -ref m2Scripts/fce.test.m2` , results was are : 
```
=========== Span-Based Correction ============
TP      FP      FN      Prec    Rec     F0.5
2       15503   4547    0.0001  0.0004  0.0002
=======================================
```
Could you please help to fix this issue? 

Kind regards 
Aiman Solyman

**Comment by chrisjbryant** [comment]
Hi Aiman,

The format of the corrected text file should be exactly the same as the original file!
E.g. if the original file has 10 sentence, the corrected file should also have 10 sentences. You then run `errant_parallel` on them as you mentioned and can then use use `errant_compare` immediately. No need to use `errant_m2` in your case.

Hope this helps. :)

**Comment by aimanmutasem** [comment]
Thank you Chris, :) 

I'm a little confused, I followed your guidelines in previous comment but the results still so poor as below,  where the BLEU score is over 80.xx.

```
=========== Span-Based Correction ============
TP      FP      FN      Prec    Rec     F0.5
7       5925    4542    0.0012  0.0015  0.0012
==============================================
```

Could you please explain this issue?

**Comment by chrisjbryant** [comment]
Heya, sorry to hear you're still having trouble. Can you post an extract of your orig_sents.txt file and cor_sents.txt file? I suspect a problem with the format of your input!

**Comment by aimanmutasem** [comment]
Really, thank you so much for kindly support, :) 
All files are uploaded in [Github](https://github.com/aimanmutasem/Apply-errant/tree/main/m2Scripts), also uploaded notebook of implementation that showed `BLEU-4: 79.517%`.

**Comment by chrisjbryant** [comment]
I just ran errant with your files and got a similar score so can confirm errant is working properly.

The problem is your `corec_sents.txt` file. One obvious issue is that everything is now in lower case, but you also probably have a bug in your system since some of the output tokens are not real words (example in bold below)!

Orig:
> As far as the organisation is concerned , an adequate number of double rooms with shower or bath have been booked for your group at the Palace Hotel , located across the main street from the train station .

Cor:
>  as far as organisation is concerned , **adquire** number of double rooms with shower or bath have been booked for your group at the **paceal** hotel located the main street from the **traination** stations .

Hopefully you can find the bug, but this is ultimately not an ERRANT issue!

**Comment by aimanmutasem** [comment]
Thank you again :) 
The lower case issue because I used BPE for tokenization. Also, I have used spacy but still face the same issue, the new file is attached.  
[corec_sents.txt](https://github.com/chrisjbryant/errant/files/7381212/corec_sents.txt)
There is an issue related to data files not the quality of the output since always I got 0.00 F1 score. If everything is working well it should get at least 5 points or lower F1 score not 0.00. 
I'll keep working on this issue and will keep you updated. 

Kind regards 
Aiman
